backend/src/handlers/items/updateBlog.js

```diff
@@ -60,7 +60,10 @@ module.exports = (app) => {
       logger.debug('Mongo connected');
 
       const query = { $set: { 'info.editorial': flag } };
-      await dbClient.db().collection('items').updateOne({ _id: blogId }, query);
+      await Promise.all([
+        dbClient.db().collection('items').updateOne({ _id: blogId }, query),
+        mongo.logAdminActions(dbClient, req.identity.userId, 'toggle editorial', blogId, { flag }),
```

Comment:
blog is managed by users not admins

Reply:
Even updating the editorial status? Doesn't that currently require admin role?